composer.json

```
@@ -11,14 +11,16 @@
 	],
 	"require": {
 		"php": ">=7.1",
+		"altis/consent": "^1.0.0",
```

For our own plugins at least we should keep the version constraint tighter so that only patch releases are allowed.
```suggestion
		"altis/consent": "~1.0.0",
```